### DISCUSSION TITLE
h5py

### DISCUSSION THREAD

**Discussion by SebGue** [Eomys/pyleecan#340] [Q&A] [answered]
Hello @BonneelP ,

there seems to be some issue with h5py (at least when loading some internal class). 
I.e. h5py is avialable in v3.2.1, which doesn't load while v2.10.0 load my file.

Maybe you could check your version and if 3.2.1 load your files.

Best regards, Sebastian

Edit: Saving seems to be okay for both versions.

**Comment by BonneelP** [comment, marked as answer]
Hello,

I thought we already talked about that but we had the same issue and we think about fixing the version to 2.10. Maybe it is worth investigating to understand what is going on, maybe 3.2.1 is more efficient so it could be worth it.

Best regards,
Pierre

**Reply by SebGue** [reply]
No this is a new issue. So fixing v2.10 seems okay as intermediate solution. (Nevertheless I won't contribute to h5, since I will go for zipped files.)
